client/components/NewBankForm.js

```diff
@@ -25,8 +25,10 @@ export default class NewBankForm extends React.Component {
     domBank() {
         return this.refs.bank.getDOMNode();
     }
-    domWebsite() {
-        return this.refs.website.getDOMNode();
+    domCustomFields() {
+        return this.state.customFields.map(function(field, index) {
+            return this.refs["customField" + index].getDOMNode();
```

Comment:
style nit: use an arrow function here, then you don't need to specify the `this` target in the map call